datasafe-rest/src/main/java/de/adorsys/datasafe/rest/config/DatasafeConfig.java

```diff
@@ -0,0 +1,62 @@
+package de.adorsys.datasafe.rest.config;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import de.adorsys.datasafe.business.api.config.DFSConfig;
+import de.adorsys.datasafe.business.api.storage.StorageService;
+import de.adorsys.datasafe.business.impl.service.DaggerDefaultDatasafeServices;
+import de.adorsys.datasafe.business.impl.service.DefaultDatasafeServices;
+import de.adorsys.datasafe.business.impl.storage.S3StorageService;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.net.URI;
+import java.security.Security;
+
+@Configuration
+public class DatasafeConfig {
+
+    private final DatasafeProperties properties;
+
+    private final AmazonS3 s3 = AmazonS3ClientBuilder.standard()
```

Comment:
It is better having this in AmazonConfig configuration

Reply:
Do you mean rename DatasafeConfig to AmazonConfig or extract just AmazonS3 init to separate bean in AmazonConfig?